graylog2-server/src/test/java/org/graylog2/shared/utilities/ExceptionUtilsTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.shared.utilities;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.fail;
+
+public class ExceptionUtilsTest {
+    @Test
+    public void formatMessageCause() throws Exception {
+
+        final NullPointerException npe = new NullPointerException();
+
+        try {
+            assertThat(ExceptionUtils.formatMessageCause(npe)).isNotBlank();
+        } catch (Exception e) {
+            fail(e.getMessage());
```

Comment:
Random nitpick: The test fail anyway if an exception was thrown and not caught in `ExceptionUtils#formatMessageCause(…)`.